examples/graphql/graphql.go

```diff
@@ -1,21 +1,23 @@
 package main
 
 import (
+	"github.com/BuxOrg/go-buxclient/xpriv"
 	"log"
 
 	"github.com/BuxOrg/go-buxclient"
-	"github.com/bitcoinschema/go-bitcoin/v2"
 )
 
 func main() {
 
-	// Example xPub
-	masterKey, _ := bitcoin.GenerateHDKey(bitcoin.SecureSeedLength)
-	// rawXPub, _ := bitcoin.GetExtendedPublicKey(masterKey)
+	// Generate keys
+	keys, resErr := xpriv.Generate()
+	if resErr != nil {
+		log.Fatalln(resErr.Error())
+	}
 
 	// Create a client
 	buxClient, err := buxclient.New(
-		buxclient.WithXPriv(masterKey.String()),
+		buxclient.WithXPriv(keys.XPub().String()),
```

Comment:
In `examples/http/http.go` you have `buxclient.WithXPriv(keys.String()),` is it correct ?

Reply:
You are right, It should be like in `examples/http/http.go`